functions/assert_supported_bolt_version.pp

```diff
@@ -6,7 +6,7 @@
 function peadm::assert_supported_bolt_version (
   # No arguments
 ) >> Struct[{ 'supported' => Boolean }] {
-  $supported_bolt_version = '>= 3.17.0 < 4.0.0'
+  $supported_bolt_version = '>= 3.17.0 < 5.0.0'
```

Comment:
Should we need to update the version in the gem file as well?

Reply:
I  checked that before raising the PR, the Gemfile doesn't pin the version.

Reply:
Looks like CI is still pulling in v3.x of the bolt gem because of the puppet_litmus dependency. We're working on getting that updated ASAP. 

If that can't be done in the next few days I've got a workaround we can add so that we're testing against Bolt 4

Reply:
Litmus updated for bolt4 now. Closing and reopening to rekick CI